src/_guides/language/effective-dart/usage.md

```diff
@@ -1072,8 +1072,10 @@ If a field doesn't depend on any constructor parameters, it can and should be
 initialized at its declaration. It takes less code and makes sure you won't
 forget to initialize it if the class has multiple constructors.
 
+<!-- Q: As of 2.9, this code no longer works.
```

Comment:
@munificent opinion?

Reply:
I mentioned this elsewhere, but to close the loop here, I think we should make `contents` non-final (assuming I'm guessing correctly at what no longer works).